packages/jellyfish-api-core/__tests__/category/blockchain.test.ts

```diff
@@ -249,7 +249,7 @@ describe('masternode', () => {
     })
   })
 
-  describe('getdifficulty', () => {
+  describe('getDifficulty', () => {
```

Comment:
This file will be removed soon

Reply:
why not remove the file in this PR??

Reply:
Let you compare the code before change and after change

Reply:
nope.. you should remove the file in this PR.. as stated in commit

Reply:
Any other comment?